config/jobs/kubernetes/sig-cloud-provider/azure/sig-azure-config.yaml

```diff
@@ -271,95 +271,3 @@ periodics:
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: Runs conformance & node conformance tests latest kubernetes master with aks-engine
     testgrid-num-columns-recent: '30'
-- interval: 24h
-  name: ci-kubernetes-e2e-capz-conformance-master
```

Comment:
replaced by periodic-cluster-api-provider-azure-conformance-with-ci-artifacts

new job added to the same dashboards